openedx/core/djangoapps/bookmarks/tests/test_api.py

```diff
@@ -177,6 +178,20 @@ def test_create_bookmark_raises_error(self, mock_tracker):
         self.assert_no_events_were_emitted(mock_tracker)
 
     @patch('openedx.core.djangoapps.bookmarks.api.tracker.emit')
+    def bookmark_more_than_limit_raise_error(self, mock_tracker):
+        """
+        Verifies that create_bookmark raises error when maximum number of units
+        allowed to bookmark per course are already bookmarked.
+        """
+        max_bookmarks = settings.MAX_BOOKMARKS_PER_COURSE
+        __, blocks, __ = self.create_course_with_bookmarks_count(max_bookmarks)
+        with self.assertNumQueries(1):
+            with self.assertRaises(BookmarksLimitReachedError):
+                api.create_bookmark(user=self.user, usage_key=unicode(blocks[-1].location))
+
+        self.assert_no_events_were_emitted(mock_tracker)
+
```

Comment:
Can you add two more checks here:
1. If another user tries to create a bookmark, it succeeds.
1. If the user tries to create a bookmark in another course that succeeds.